.env.production

```diff
@@ -1 +0,0 @@
-REACT_APP_GITHUB_API_URL=https://databraid-github-api-lb-856903072.us-west-1.elb.amazonaws.coma
```

Comment:
Why remove this?

Reply:
there is absolutely no need to remove this and i didn't touch it. apparently i'm continuing to see unicorns- somethiing isn't working with ```git pull upstream dev```

Reply:
good catch @HamidAghdaee

Reply:
I'm not sure @Gwillison415 but I fixed this for you. You should do `git fetch upstream` and then `git rebase upstream/dev` from now on.